ros2launch_security/option/security.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2020 Canonical, Ltd.
+# Copyright 2021 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import pathlib
+from tempfile import TemporaryDirectory
+from typing import Optional
+from typing import Tuple
+
+import launch
+from ros2launch.option import OptionExtension
+import sros2.api._keystore
+
+
+class NoKeystoreProvidedError(Exception):
+    """Exception raised when a keystore is not provided."""
+
+    def __init__(self):
+        super().__init__(('--no-create-keystore was specified and '
+                          'no keystore was provided'))
+
+
+class NonexistantKeystoreError(Exception):
```

Comment:
@gbiggs typo:
```suggestion
class NonexistentKeystoreError(Exception):
```
? Or perhaps I'm unaware of this different spelling.

Reply:
Nope, just a typo! Good catch.